packages/grid_client/tests/modules/zmount.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Zmount } from "../../src";
+
+let zmount: Zmount;
+
+beforeEach(() => {
+  zmount = new Zmount();
+});
+describe("Zmount module", () => {
+  test("Zmount instance is of type Zmount.", () => {
+    expect(zmount).toBeInstanceOf(Zmount);
+  });
+
+  test("Min value for size.", () => {
+    const size = 100 * 1025 ** 2;
+
+    zmount.size = size;
+
+    const result = () => zmount.challenge();
+
+    expect(result).toThrow();
+  });
+
+  test("Max value for size.", () => {
+    const size = 100 * 1025 ** 4;
+
+    zmount.size = size;
+
+    const result = () => zmount.challenge();
+
+    expect(result).toThrow();
+  });
+
+  test("Size doesn't accept decimal value.", () => {
```

Comment:
How about test("Size doesn't accept strings") ?

Reply:
https://github.com/threefoldtech/tfgrid-sdk-ts/blob/7148518e9faa2ffdc098df54993c0c243175d6ac/packages/grid_client/src/zos/zmount.ts#L7

Typescript won't let me run the code as the size is of the type number.